### PR TITLE
✨ marcuscodes.me

### PR DESCRIPTION
**Pull request by geenva**

### CODE REVIEW

File: websites/M/marcuscodes.me/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "id": "457805013474082817"
+  },
+  "service": "marcuscodes.me",
+  "description": {
+    "en": "The personal website of geneva/marcus."
+  },
+  "url": "marcuscodes.me",
+  "version": "1.0.2",
+  "logo": "https://cdn.discordapp.com/attachments/740930314121117716/782533306481639424/unknown.png",
+  "thumbnail": "https://cdn.discordapp.com/attachments/740930314121117716/782533306481639424/unknown.png",
```
Comment (Bas950) [inline websites/M/marcuscodes.me/dist/metadata.json:13]: upload to imgur

File: websites/M/marcuscodes.me/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "author": {
+    "name": "geneva",
+    "id": "457805013474082817"
+  },
+  "service": "marcuscodes.me",
+  "description": {
+    "en": "The personal website of geneva/marcus."
+  },
+  "url": "marcuscodes.me",
+  "version": "1.0.2",
```
Comment (Bas950) [inline websites/M/marcuscodes.me/dist/metadata.json:11]: v1.0.0

File: websites/M/marcuscodes.me/presence.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const presence = new Presence({
+  clientId: "782532120047779850"
+});
+
+const timeStamp = Math.floor(Date.now() / 1000);
+const page = window.location.href;
```
Comment (Bas950) [inline websites/M/marcuscodes.me/presence.ts:6]: merge consts

File: websites/M/marcuscodes.me/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "logo": "https://cdn.discordapp.com/attachments/740930314121117716/782533306481639424/unknown.png",
+  "thumbnail": "https://cdn.discordapp.com/attachments/740930314121117716/782533306481639424/unknown.png",
+  "color": "#00008B",
+  "tags": [
+    "geneva", "marcus", "personal website"
+  ],
```
Comment (Bas950) [inline websites/M/marcuscodes.me/dist/metadata.json:17]: tags cannot have spaces

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by geenva** [comment]
> upload proof of it working

https://i.imgur.com/HOpJiOC.png

File: websites/M/marcuscodes.me/presence.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
+const presence = new Presence({
+  clientId: "782532120047779850"
+});
+const timeStamp = Math.floor(Date.now() / 1000);
+const page = window.location.href;
```
Comment (Bas950) [inline websites/M/marcuscodes.me/presence.ts:5]: Merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/marcuscodes.me/presence.ts (hunk 5 above)
Comment (geenva) [reply websites/M/marcuscodes.me/presence.ts:5]: sorry for my stupidity, what does that mean

Comment (ririxidev) [reply websites/M/marcuscodes.me/presence.ts:5]: e.g.: ```const d = f(),
g = h();```

Comment (geenva) [reply websites/M/marcuscodes.me/presence.ts]: i hope this works

Comment (ririxidev) [reply websites/M/marcuscodes.me/presence.ts]: image
you didnt change anything

File: websites/M/marcuscodes.me/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    "id": "457805013474082817"
+  },
+  "service": "marcuscodes.me",
+  "description": {
+    "en": "The personal website of geneva/marcus."
+  },
+  "url": "marcuscodes.me",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/DSs0sfV.png",
+  "thumbnail": "https://i.imgur.com/DSs0sfV.png",
```
Comment (ririxidev) [inline websites/M/marcuscodes.me/dist/metadata.json:13]: this should be banner or promo img of site.

File: websites/M/marcuscodes.me/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "geneva",
+    "id": "457805013474082817"
+  },
+  "service": "marcuscodes.me",
+  "description": {
+    "en": "The personal website of geneva/marcus."
+  },
+  "url": "marcuscodes.me",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/DSs0sfV.png",
```
Comment (ririxidev) [inline websites/M/marcuscodes.me/dist/metadata.json:12]: Change this logo to something that is not only circle with red colors.

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/M/marcuscodes.me/presence.ts [hunk 8]
```diff
@@ -3,13 +3,14 @@ const presence = new Presence({
...
 
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "circlegrad",
     startTimestamp: timeStamp
   };
-  const page = window.location.href;
+
   if (page == "https://marcuscodes.me/") {
```
Comment (Bas950) [inline websites/M/marcuscodes.me/presence.ts]: double check if it still works with this change

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/marcuscodes.me/presence.ts (hunk 8 above)
Comment (geenva) [reply websites/M/marcuscodes.me/presence.ts]: yes

File: websites/M/marcuscodes.me/dist/metadata.json (hunk 7 above)
Comment (geenva) [reply websites/M/marcuscodes.me/dist/metadata.json:12]: That's the website logo.

Comment (geenva) [reply websites/M/marcuscodes.me/dist/metadata.json:12]: Should I change it to something else other than the favicon?

Comment (Bas950) [reply websites/M/marcuscodes.me/dist/metadata.json:12]: I mean... yes... a red dot aint really a logo imo

**Comment by ririxidev** [comment]
After talking with @Bas950 we decided to not merge this presence since it is low quality one. If you want more details just mention us/or me/or bas.

**Pull request closed without merging**